Data.js

```diff
@@ -0,0 +1,156 @@
+const data = [
+  {
+    id: 1,
+    name: 'Tonic',
+    tags: [
+      { name: 'html', shouldShowOnMobile: true },
+      { name: 'css', shouldShowOnMobile: true },
+      { name: 'javascript', shouldShowOnMobile: true },
+      { name: 'Ruby on rails', shouldShowOnMobile: false },
+    ],
```

Comment:
- [ ] Kindly add live link and source link properties to each object. Since it is a requirement for this milestone and you are gonna need it when you add actual projects to your portfolio.